ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-bgp-evpn-mpls.cfg

```diff
@@ -6,11 +6,13 @@ interface Management1
 !
 router bgp 65101
    router-id 192.168.255.3
+   timers bgp min-hold-time 5 send-failure hold-time 65
    no bgp default ipv4-unicast
```

Comment:
I my test the timers comes after the no bgp default ipv4-unicast

Reply:
+1

```eos
+router bgp 65101
+   router-id 192.168.255.3
+   no bgp default ipv4-unicast
+   timers bgp min-hold-time 5 send-failure hold-time 65
+   distance bgp 20 200 200
+   graceful-restart restart-time 300
+   graceful-restart
+   maximum-paths 2 ecmp 2
+   neighbor default send-community standard extended large
+   neighbor EVPN-OVERLAY-PEERS peer group
+   neighbor EVPN-OVERLAY-PEERS remote-as 65001
+   neighbor EVPN-OVERLAY-PEERS update-source Loopback0
```

Reply:
Yes the ordering is same in the template, this mismatch in the ordering of this file is because these commands are defined under `bgp_defaults` keys, which is below timers in the template.

We can see the correct ordering in `router-bgp-base.cfg`

Reply:
https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-evpn-mpls.yml#L4

Reply:
ok thank you for clearing that up.